tests/unit/lms/services/digest_test.py

```diff
@@ -158,30 +158,50 @@ def svc(self, db_session, h_api, sender, email_unsubscribe_service):
 class TestDigestContext:
     def test_instructor_digest(self, db_session, make_instructor):
         courses = factories.Course.create_batch(2)
-        instructor, learner = factories.User.create_batch(2)
+        instructor, learner_1, learner_2 = factories.User.create_batch(3)
```

Comment:
Making this test case more interesting: it now covers both a course with multiple annotators, and a course with multiple annotations by the same annotator (should be deduplicated when counting the number of annotators)